apps/wallet/src/ui/app/shared/transaction-summary/cards/ObjectChanges.tsx

```diff
@@ -34,146 +34,222 @@ function ChevronDown({ expanded }: { expanded: boolean }) {
     );
 }
 
-function ObjectDetail({
-    objectId,
-    objectType,
+export function ObjectDetail({
+    change,
+    display,
 }: {
-    objectId: string;
-    objectType: string;
+    change: SuiObjectChangeWithDisplay;
+    ownerKey: string;
+    display?: boolean;
 }) {
-    const [expanded, setExpanded] = useState(false);
-
+    if (
+        is(change, SuiObjectChangeTransferred) ||
```

Comment:
As a small aside, is it possible to do this through property-checking instead? I actually want to deprecate `is` because I want to move to a world where folks property-check instead of doing it like this (because this depends on runtime schemas that are shipped with the SDK which I want to go away).